packages/composite-checkout/src/components/checkout.js

```diff
@@ -249,13 +259,6 @@ function ReviewOrderStep( { isActive, isComplete, ReviewContent } ) {
 						<CheckoutReviewOrder isActive={ isActive } />
 					)
 				}
-				stepSummary={
```

Comment:
Another way to do this would be to change `ReviewContent` and `CheckoutReviewOrder` to return `null` if `summary` is set. That leaves the choice up to the component that implements the step, so if the host page wants to show something it has that option.

Reply:
> Another way to do this would be to change ReviewContent and CheckoutReviewOrder to return null if summary is set.

Do you think this would still be necessary if we give folks the option to use the CheckoutHeader in the order summary?